sciLifeLab_utils/run_validation.py

```diff
@@ -76,7 +76,7 @@ def main(args):
             sample_YAML.close
 
             # Run the job
-            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.1b1\n"]
+            extramodules = ["module load samtools\nmodule load bwa\nmodule load BUSCO/1.22\nsource $BUSCO_SETUP\n"]
```

Comment:
`source $BUSCO_SETUP` is it something intended for `irma` ? Hope it will not affect the jobs while running in `milou-b`